src/time.ts

```diff
@@ -76,3 +76,16 @@ export function parseDuration(input: string): number {
 
   return total;
 }
+
+/**
+ * sleep waits for a specified duration in milliseconds
+ *
+ * @param ms. Duration in milliseconds to sleep.
+ */
+export async function sleep(ms = 0): Promise<void> {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve();
+    }, Number(ms));
```

Comment:
Should the type be explicit on the function instead?

Reply:
Linter threw an error, I will remove the Number fn